docs/install/README.md

```diff
@@ -10,13 +10,16 @@ ko apply -f ./config
 
 Applying the `config` dir will:
 
-- create a new namespace, `cloud-run-events`.
-- install new CRDs,
+- Create a new namespace, `cloud-run-events`.
+- Install new CRDs:
+  - `channels.pubsub.cloud.run`
   - `pullsubscriptions.pubsub.cloud.run`
-- install a controller to operate on the new resources,
+  - `topics.pubsub.cloud.run`
+- Install a controller Deployment to operate on the new resources,
   `cloud-run-events/cloud-run-events-controller`.
-- install a new service account, `cloud-run-events/cloud-run-events-controller`.
-- provide RBAC for that service account.
+- Install a webhook Deployment to validate and mutate the new resources, `cloud-run-events/cloud-run-events-webhook`.
```

Comment:
Note that this is actually incorrect, the Deployments are named `webhook` and `controller`. I think we should change the names of the Deployments to match these docs. Thougths?

Reply:
I want them to be called `webhook` and `controller`. Can you update the doc?